src/Components/Navigation/SidePhoto.js

```diff
@@ -1,17 +1,16 @@
-import { Show, Image, Box } from "@chakra-ui/react";
+import { Show, Image, Box, useColorModeValue } from "@chakra-ui/react";
 
 const SidePhoto = () => {
+  const photo = useColorModeValue("https://i.ibb.co/Q6JNRKg/light-mode.jpg", "https://i.ibb.co/sQ5S8zy/dark-mode.jpg")
   return (
     <Show above="xl">
-      <Box
-        mt="-16"
-        right={0}
-        zIndex={-2}
-        pos={"fixed"}
-        borderLeft={"1px"}
-        borderColor={"gray.700"}
-      >
-        <Image src="https://i.ibb.co/FqbjLw6/dark-banner-2.png"></Image>
+      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"}  
+      h="100%" 
+      bgSize="100%" 
+      bgImage={`url(${photo})`}
+      backgroundRepeat="repeat"
+      bgPosition="top">
+      X // is this a typo?
```

Comment:
one more thing, let's delete this line? i accidentally moved it outside of the <Box> tag, but there was a seemingly random "X" in there to begin with.

```suggestion

```

Reply:
My bad- thought it was taken out in the revision, but absolutely